packages/theme-predictive-search-component/README.md

```diff
@@ -0,0 +1,57 @@
+## Description
+
+A component that controls the state of a predictive search UI.
```

Comment:
As discussed IRL, I think this needs more details and also make reference to the predictive search API package

Reply:
We can add more info in a follow-up PR. For now, the API has changed, so we are misinforming developers.